### PR TITLE
feat(Task-18): As an User, I want a descriptive browser tab information

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

**Comment by jLagosCarrera** [comment]
Same as #25, when #24 is approved, add pt translations

File: src/app/app.title.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default ['$transitions', '$window', '$q', ($transitions, $window, $q) => {
+    $transitions.onSuccess({}, (transition) => {
+        let title = "";
+        const tokens = transition.getResolveTokens();
+        const resolves = {};
+        const promises = tokens.map(function (token) {
+            var resolved = transition.injector().get(token);
```
Comment (gbtec-ext-jorgevillar) [inline src/app/app.title.js]: 'var' detected!

File: src/app/app.title.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export default ['$transitions', '$window', '$q', ($transitions, $window, $q) => {
+    $transitions.onSuccess({}, (transition) => {
+        let title = "";
```
Comment (gbtec-ext-jorgevillar) [inline src/app/app.title.js]: use single quotes

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

File: src/app/app.title.js (hunk 1 above)
Comment (eloydotco) [inline src/app/app.title.js]: can this be a const??

File: src/app/app.title.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        const tokens = transition.getResolveTokens();
+        const resolves = {};
+        const promises = tokens.map(function (token) {
```
Comment (eloydotco) [inline src/app/app.title.js]: you can use an arrow function here

File: src/app/app.title.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            var resolved = transition.injector().get(token);
+            resolves[token] = resolved;
+            return resolved;
+        });
+        $q.all(promises).then(function () {
```
Comment (eloydotco) [inline src/app/app.title.js]: you can use an arrow function here

**Review by eloydotco — APPROVED** [review]

File: config/webpack.config.dev.js [hunk 5]
```diff
@@ -20,6 +21,7 @@ module.exports = {
             template: './src/index.html',
             filename: './index.html'
         }),
+        new FaviconsWebpackPlugin('./src/resources/img/favicon.png'),
```
Comment (jLagosCarrera) [inline config/webpack.config.dev.js:24]: Configure for development and production

**Pull request merged**